session/session.go

```diff
@@ -1275,6 +1275,7 @@ const loadCommonGlobalVarsSQL = "select HIGH_PRIORITY * from mysql.global_variab
 	variable.TiDBHashAggPartialConcurrency + quoteCommaQuote +
 	variable.TiDBHashAggFinalConcurrency + quoteCommaQuote +
 	variable.TiDBBackoffLockFast + quoteCommaQuote +
+	variable.TiDBDeferConstraintCheck + quoteCommaQuote +
```

Comment:
I test the behavior, if this global variable do not exist in the underlying TiKV (sadly it holds), `sessionVar.DeferConstraintCheck` will not be initialized to 'true'.

```
mysql> select HIGH_PRIORITY * from mysql.global_variables where variable_name in ('a_new_add_global', 'tidb_index_join_batch_size', 'tidb_index_lookup_size');
+----------------------------+----------------+
| VARIABLE_NAME              | VARIABLE_VALUE |
+----------------------------+----------------+
| tidb_index_lookup_size     | 20000          |
| tidb_index_join_batch_size | 25000          |
+----------------------------+----------------+
2 rows in set (0.01 sec)
```

I insist that 'false' is a much better default value.